authors/apps/articles/urls.py

```diff
@@ -20,7 +19,5 @@
     path('articles/<slug>/like',LikeArticleAPIView.as_view()),
     path('articles/<slug>/dislike',DislikeArticleAPIView.as_view()),
     path('articles/<slug>/favorite',FavoriteArticleAPIView.as_view()),
-    path('my/articles/', ListAuthorArticlesAPIView.as_view()),
```

Comment:
Help me understand why these URLs are being removed.

Reply:
Those urls are no longer needed. The functionality they were providing is no longer necessary.